gammapy/stats/fit_statistics.py

```diff
@@ -155,18 +133,40 @@ def wstat(mu_signal, n_on, n_off, alpha):
 
     References
     ----------
-    * Statistics page :ref:`stats`
     * `Habilitation M. de Naurois, p. 141
       <http://inspirehep.net/record/1122589/files/these_short.pdf>`_
     * `XSPEC page on Poisson data with Poisson background
       <http://heasarc.nasa.gov/xanadu/xspec/manual/XSappendixStatistics.html>`_
     """
+    # Note: This is equivalent to what's defined on the XSPEC page under the
+    # following assumptions
+    # t_s * m_i = mu_signal
+    # t_b * m_b = mu_background
+    # t_s / t_b = alpha
 
-    mu_signal = np.asanyarray(mu_signal, dtype=np.float64)
     n_on = np.asanyarray(n_on, dtype=np.float64)
     n_off = np.asanyarray(n_off, dtype=np.float64)
     alpha = np.asanyarray(alpha, dtype=np.float64)
+    mu_signal = np.asanyarray(mu_signal, dtype=np.float64)
+   
+    mu_background = _get_wstat_background(n_on, n_off, alpha, mu_signal)
+
+    
+    term1 = mu_signal + (1 + alpha) * mu_background 
+    term2 = - n_on * np.log(mu_signal + alpha * mu_background)
+    term3 = - n_off * np.log(mu_background) 
+    
+    stat = 2 * (term1 + term2 + term3)
 
+    if extra_terms:
+        term = _get_wstat_extra_terms(n_on, n_off)
```

Comment:
Maybe not put `term` temp variable here?
Seems pointless.
(I do it all the time, but only if lines get too long or if it will give simpler tracebacks because fewer computations happen on the same line. Here there's no such advantage in introducing a temp variable `term`)
Just put:
```python
stat += _get_wstat_extra_terms(n_on, n_off)
```